"gzip -cn: gzip: command not found" error when installing any git-based package

On latest vcpkg HEAD (6633d421b7f53ba6faadb8ee85ef806718dfd64b):

```
PS C:\Users\edechamps\Documents\vcpkg> .\vcpkg install fdlibm
The following packages will be built and installed:
    fdlibm[core]:x86-windows
Starting package 1/1: fdlibm:x86-windows
Building package fdlibm[core]:x86-windows...
-- Fetching https://android.googlesource.com/platform/external/fdlibm...
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:56 (message):
    Command failed: C:/Users/edechamps/Documents/vcpkg/downloads/tools/git-2.19.1-windows/cmd/git.exe;archive;FETCH_HEAD;-o;C:/Users/edechamps/Documents/vcpkg/downloads/temp/fdlibm-59f7335e4dd8275a7dc2f8aeb4fd00758fde37ac.tar.gz
    Working Directory: C:/Users/edechamps/Documents/vcpkg/downloads/git-tmp
    See logs for more information:
      C:\Users\edechamps\Documents\vcpkg\buildtrees\fdlibm\git-archive-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_from_git.cmake:104 (vcpkg_execute_required_process)
  ports/fdlibm/portfile.cmake:3 (vcpkg_from_git)
  scripts/ports.cmake:71 (include)


Error: Building package fdlibm:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: fdlibm:x86-windows
  Vcpkg version: 2018.10.20-nohash

Additionally, attach any relevant sections from the log files above.
```

`C:\Users\edechamps\Documents\vcpkg\buildtrees\fdlibm\git-archive-err.log`:

```
gzip -cn: gzip: command not found
```

The same issue occurs on any package that uses `vcpkg_from_git()`.

Looks like `gzip` is missing. Maybe vcpkg should install it automatically?

--- a/scripts/vcpkgTools.xml
+++ b/scripts/vcpkgTools.xml
@@ -22,11 +22,11 @@
         <archiveName>cmake-3.12.4-Linux-x86_64.tar.gz</archiveName>
     </tool>
     <tool name="git" os="windows">
-        <version>2.19.1</version>
+        <version>2.20.0</version>
         <exeRelativePath>cmd\git.exe</exeRelativePath>
-        <url>https://github.com/git-for-windows/git/releases/download/v2.19.1.windows.1/PortableGit-2.19.1-32-bit.7z.exe</url>
-        <sha512>50e11ba1190bca0210cf1aac4a1a10acddc8cff2f15ac4d036c1fcbab816ee35089b6115554e7d346dfb6f37003dc5da4bea396f3db4dae9b2f2bdd23fd0d876</sha512>
-        <archiveName>PortableGit-2.19.1-32-bit.7z.exe</archiveName>
+        <url>https://github.com/git-for-windows/git/releases/download/v2.20.0.windows.1/PortableGit-2.20.0-32-bit.7z.exe</url>
+        <sha512>81647a87df9fde0945ef597cb1cafd8f5f42859da89e9b1db55222a261407bc16bdcc0cf1e86e315697f0981832fe10fc02845cad4b4c82ea64bbd218aec6a49</sha512>
+        <archiveName>PortableGit-2.20.0-32-bit.7z.exe</archiveName>
     </tool>
     <tool name="git" os="linux">
         <version>2.7.4</version>

--- a/scripts/vcpkgTools.xml
+++ b/scripts/vcpkgTools.xml
@@ -24,9 +24,9 @@
     <tool name="git" os="windows">
         <version>2.19.1</version>
         <exeRelativePath>cmd\git.exe</exeRelativePath>
-        <url>https://github.com/git-for-windows/git/releases/download/v2.19.1.windows.1/MinGit-2.19.1-32-bit.zip</url>
-        <sha512>8a6d2caae2cbaacee073a641cda21465a749325c0af620dabd0e5521c84c92c8d747caa468b111d2ec52b99aee2ee3e6ec41a0a07a8fff582f4c8da568ea329e</sha512>
-        <archiveName>MinGit-2.19.1-32-bit.zip</archiveName>
+        <url>https://github.com/git-for-windows/git/releases/download/v2.19.1.windows.1/PortableGit-2.19.1-32-bit.7z.exe</url>
+        <sha512>50e11ba1190bca0210cf1aac4a1a10acddc8cff2f15ac4d036c1fcbab816ee35089b6115554e7d346dfb6f37003dc5da4bea396f3db4dae9b2f2bdd23fd0d876</sha512>
+        <archiveName>PortableGit-2.19.1-32-bit.7z.exe</archiveName>
     </tool>
     <tool name="git" os="linux">
         <version>2.7.4</version>

